task6_Shai_Axelrod/Task06/src/DistanceCalc.cpp

```diff
@@ -0,0 +1,43 @@
+#include "DistanceClac.h"
+#include <stdexcept>
+
+// euclidean Distance
+static float  euclideanDistance(Location loc1, Location loc2) {
```

Comment:
there is std::function in the STL library that you can use and not a pointer to function